src/BloomExe/Edit/EditingModel.cs

```diff
@@ -509,6 +515,29 @@ void OnPageSelectionChanged(object sender, EventArgs e)
 			GC.Collect();//i put this in while looking for memory leaks, feel free to remove it.
 		}
 
+		private void CheckForUnsavedOrigamiAndSave()
```

Comment:
It's not obvious either that what this method is looking for identifies unsaved origami, or that what it does somehow saves it. If the method name is right, I think some more explanation is in order.